code-the-dream-react/src/App.js

```diff
@@ -1,38 +1,76 @@
-import * as React from 'react';
 import { useEffect } from 'react';
+import { useState } from 'react';
 import { TodoList } from './TodoList.js';
 import { AddTodoForm } from './AddTodoForm.js';
- 
-function useSemiPersistentState() {
-  const [todoList, setTodoList] = React.useState(() => {
-    const savedTodoList = localStorage.getItem('savedTodoList');
-    return savedTodoList ? JSON.parse(savedTodoList) : [];
-  });
+
+function App() {
+  const [todoList, setTodoList] = useState([]);
+  const [isLoading, setIsLoading] = useState(true)
+
+  async function fetchData() {    // What's the difference (if any) between initiating an async function the way I did it vs const fetchData = async() =>
```

Comment:
the only difference is syntax, both ways achieve the exact same thing. the ES6 syntax is "newer"